demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -2068,6 +2068,8 @@ def __init__(self, build_context: BuildContext, server_private_ip: str, tunnel_p
         self.executed_in_current_round: Set[str] = set()
         self.prev_system_conf: dict = {}
         self.use_retries_mechanism: bool = use_retries_mechanism
+        if IS_XSIAM:
+            self.check_if_can_create_manual_alerts(self.client)
```

Comment:
```suggestion
            self.check_if_can_create_manual_alerts()
```
use self.client in the function